backend/apps/pr_review_bot/services/code_analyzer.py

```
@@ -0,0 +1,177 @@
+            errors = style_checker.check_all()
+            
+            if errors:
+                self.issues.append({
+                    'type': 'style',
+                    'severity': 'warning',
+                    'title': 'Style issues detected',
+                    'description': f'Found {errors} style violations',
+                    'file_path': file_path,
+                    'suggestion': 'Run black or autopep8 to fix style issues'
+                })
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)